README.md

```diff
@@ -8,54 +8,38 @@ Android POC for www.covid-watch.org
 - Offer a seamless UX to complete all of the above!
 
-The current version of the Figma we're working towards: https://www.figma.com/file/0uL6s79o21jwUFZz16Dr8b/Covid-Watch---App-v1.8?node-id=0%3A1
+The current version of the Figma we're working towards: (https://www.figma.com/file/I4OOjT4VhsSberDIAZTkcH/Covid-Watch---App-v2-(dev)?node-id=0%3A1
+
 
 ## Setup
 
 Clone this repo from the `develop` branch:
 
-```
 git clone git@github.com:covid19risk/covidwatch-android.git
-```
-
-We are using tracking TCN's project as a submodule for now tracking the develop branch, so don't forget to init and fetch.
-
-First time:
-
-```
-$ git submodule update --init --recursive --remote
-```
-
-To Update:
-
-```
-git submodule update --remote
-```
 
 Open the project in Android Studio. Install onto a phone of yours with the `app` configuration, and you're free to explore the app! Its optimal to install on 2 phones as much of the behavior of the app depends on 2 phones interacting.
 
 **Note:** You cannot run this app on an emulator! We are dependent on Bluetooth being on and active, and most standard Android emulators do not have Bluetooth drivers.
 
 ## Looking to contribute?
 
-- Run on your own device to explore the UX. Look at the [Figma](https://www.figma.com/file/0uL6s79o21jwUFZz16Dr8b/Covid-Watch---App-v1.8?node-id=0%3A1) for what the UX should look like. If you have any feedback/find any problems, create an issue!
-- Look at https://github.com/orgs/covid19risk/projects/1 for existing issues. If you see something you want to work on, assign yourself to it, set it to in progress, and make a PR to the `develop` branch.
+- Run on your own device to explore the UX. Look at the [Figma](https://www.figma.com/file/I4OOjT4VhsSberDIAZTkcH/Covid-Watch---App-v2-(dev)?node-id=0%3A1) for what the UX should look like. If you have any feedback/find any problems, create an issue!
+- Look at https://blog.google/documents/68/Android_Exposure_Notification_API_documentation_v1.2.pdf for existing issues. If you see something you want to work on, assign yourself to it, set it to in progress, and make a PR to the `develop` branch.
 
 ## FAQ
 
 What is the anonymous protocol for communication between phones? How does it work and who designed it?
 
-Covid Watch uses Temporary Contact Numbers, a decentralized, privacy-first contact tracing protocol developed by the [TCN Coalition](https://tcn-coalition.org/). This protocol is built to be extensible, with the goal of providing interoperability between contact tracing applications. You can read more about it on their [Github](https://github.com/TCNCoalition/TCN).
+Covid Watch uses Google/Android Exposure Notification, a decentralized, privacy-first contact tracing API implemented in Google and Apple phone operation systems. You can read more about it in the Google design document at https://blog.google/documents/68/Android_Exposure_Notification_API_documentation_v1.3.1.pdf
```

Comment:
This PDF gets updates from time to time so it will more safe to give a link to the overview:
https://www.google.com/covid19/exposurenotifications/